src/main/java/com/backbase/ct/bbfuel/input/TransactionsReader.java

```diff
@@ -25,7 +25,20 @@ public TransactionsPostRequestBody loadSingle(String externalArrangementId) {
         return getRandomFromList(load(globalProperties.getString(CommonConstants.PROPERTY_TRANSACTIONS_DATA_JSON)))
                 .withId(UUID.randomUUID().toString())
                 .withArrangementId(externalArrangementId)
-                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)))
+                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)));
+    }
+
+    /**
+     * To be able to retrieve check images for a transaction, we need at least one of the transactions with
+     * with transactions id = TRANS0000000000001
+     * We need to set the date of the transaction to today so that we can easly find that transaction and test it.
+     */
+    public static TransactionsPostRequestBody loadSingleWithCheckImages(String externalArrangementId) {
+        TransactionsPostRequestBody defaultBody = this.loadSingle(externalArrangementId);
+        return defaultBody.withId("TRANS0000000000001")
+                .withCreditDebitIndicator(DBIT)
+                .withBookingDate(new Date())
+                .withValueDate(new Date())
                 .withCheckSerialNumber(new BigDecimal(generateRandomNumberInRange(1, 99999999)));
```

Comment:
maybe better move this to the hardcoded check transaction as well

Reply:
i did not wanna move this to the json file because we are still busy with the check images implementation, and i think that these check numbers also need to be also unique.

Perhaps @wendybackbase  can confirm this?

Reply:
Check numbers don't have to be unique.

Reply:
ok great then i can add to the json

Reply:
done